src/main/io/ledstrip.c

```diff
@@ -596,6 +598,54 @@ static void applyLedWarningLayer(bool updateNow, uint32_t *timer)
     }
 }
 
+static void applyLedBatteryLayer(bool updateNow, uint32_t *timer)
+{
+    static bool batteryFlash = false;
+
+    batteryState_e battState = getBatteryState();
```

Comment:
Update battState only when  updateNow is true (using `static uint8_t` for state)